Source/Diagnostics/FullDiagnostics.cpp

```diff
@@ -407,9 +408,13 @@ FullDiagnostics::InitializeFieldFunctors (int lev)
     // Species index to loop over species that dump rho per species
     int i = 0;
 
-    m_all_field_functors[lev].resize( m_varnames.size() );
+    int nvar = m_varnames_fields.size();
+    int nspec = m_pfield_species.size();
+    int ntot = nvar + m_pfield_varnames.size() * nspec;
```

Comment:
Can these also be `const`?
```suggestion
    const int nvar = m_varnames_fields.size();
    const int nspec = m_pfield_species.size();
    const int ntot = nvar + m_pfield_varnames.size() * nspec;
```